src/latexify/codegen/function_codegen_test.py

```diff
@@ -40,6 +40,25 @@ def test_visit_functiondef_use_signature() -> None:
     assert FunctionCodegen(use_signature=True).visit(tree) == latex_with_flag
 
 
+def test_visit_functiondef_ignore_docstring() -> None:
+    tree = ast.FunctionDef(
+        name="f",
+        args=ast.arguments(
+            args=[ast.arg(arg="x")],
+            kwonlyargs=[],
+            kw_defaults=[],
+            defaults=[],
+        ),
+        body=[
+            ast.Expr(ast_utils.make_constant("""docstring""")),
+            ast.Return(value=ast.Name(id="x", ctx=ast.Load())),
+        ],
+        decorator_list=[],
+    )
+    latex_with_flag = r"\mathrm{f}(x) = x"
+    assert FunctionCodegen().visit(tree) == latex_with_flag
```

Comment:
We don't have any "flag" here.

```suggestion
    latex = r"\mathrm{f}(x) = x"
    assert FunctionCodegen().visit(tree) == latex
```